Backpack/Calendar/Classes/BPKCalendarStickyHeader.m

```diff
@@ -101,6 +104,7 @@ - (void)setMonth:(NSDate *)month {
                                                                    content:self.titleLabel.text
                                                                  textColor:appearance.headerTitleColor];
     self.titleLabel.attributedText = monthText;
+    self.titleLabel.accessibilityLabel = [[BPKCalendarStickyHeader formatter] stringFromDate:month];
```

Comment:
Change the month label from "May" to "May, 2023". This is because the floating blue pill is ignored by screen readers (and only shows up when in 2024).

Adding the year adds a bit 'quality of life'